patches/chrome-android-java-AndroidManifest.xml.patch

```diff
@@ -63,7 +63,15 @@ index 8cd1fb84e55c6fb971db3441fa41bd7460ed738e..369cd188698f05263ce2cbe20971049a
          android:icon="@drawable/ic_launcher"
          android:roundIcon="@drawable/ic_launcher_round"
          android:label="{% block application_label %}@string/app_name{% endblock %}"
-@@ -1294,6 +1309,7 @@ android:value="true" />
+@@ -175,6 +190,7 @@ by a child template that "extends" this file.
+         <!-- ARCore APK integration -->
+         <!-- This tag indicates that this application optionally uses ARCore. -->
+         <meta-data android:name="com.google.ar.core" android:value="optional" />
++        <meta-data android:name="com.google.ar.core.min_apk_version" android:value="190603076"/><activity android:name="com.google.ar.core.InstallActivity" android:configChanges="keyboardHidden|orientation|screenSize" android:excludeFromRecents="true" android:exported="false" android:launchMode="singleTop" android:theme="@android:style/Theme.Material.Light.Dialog.Alert" />
```

Comment:
Please check a comment inside an issue why do we need that https://github.com/brave/brave-browser/issues/9345#issuecomment-616925528